Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v2/Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "==5.0.1"
+pytest-mock = "*"
+requests-mock = "*"
+pytest-asyncio = "*"
+
+[packages]
```

Comment:
Aren't we missing the pycti here?

Reply:
@daryakoval I don't see we added this ☝️

Reply:
added